pywbem/cim_operations.py

```diff
@@ -643,6 +661,24 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
             Note that not all situations can be handled within this timeout, so
             for some issues, operations may take longer before raising an
             exception.
+
+          use_pull_operations (:class:`py:bool`):
+            Controls the use of pull operations in any `Iter...()` methods.
+            The default value (`None`) will work on any WBEM server whether or
+            not it supports pull operations; first the pull operation is
+            attempted and if the WBEM server does not support pull operations,
+            the corresponding instance variable is set to `False` and the
+            traditional (non-pull) operation will be executed for the current
+            operation and all subsequent Iter...()operations for this
+            WBEM Connection.
```

Comment:
The paragraph above is not explicit about the fact that the instance variable is set to True is the server supports pull operations. I suggest to replace it with this text:

```
            Controls the use of pull operations in any `Iter...()` methods.
            The default value (`None`) will work on any WBEM server whether or
            not it supports pull operations.

            `None` means that first the pull operation is attempted. If that
            succeeds, corresponding instance variable is set to `True`. If the
            WBEM server does not support pull operations, the corresponding
            instance variable is set to `False` and the traditional (non-pull)
            operation will be executed.
```